docs/development/getting_started.rst

```diff
@@ -1,6 +1,14 @@
 Getting Started
 ===============
 
+.. note:: SecureDrop maintains two branches of documentation, ``stable`` and
+          ``latest``, the former being the default on our Read the Docs powered
+          site. ``stable`` is built from our latest signed git tag, while
```

Comment:
I would add "and corresponds to the latest stable release" or something to that effect, to help the reader understand why there are separate versions of the documentation and when they might want to use one instead of the other.